### DISCUSSION TITLE
Successful downloads are being flagged `Hidden`

### DISCUSSION THREAD

**Discussion by daFreeMan** [nzbgetcom/nzbget#382] [General]
I'm running V24.2 and all of a sudden, I've got a lot of successful downloads being flagged as `Hidden`. 

I recently changed paths for where nzbGet is saving incomplete and complete files, but I don't believe I changed anything else. This happened several months ago (don't recall the version) as well, then went back to working normally.

Here is a normal view:
![image](https://github.com/user-attachments/assets/bc78e08f-ba7a-48d1-9263-7bb3e9c3f5e2)

And here it is showing all the hidden successes:
![image](https://github.com/user-attachments/assets/9666947a-f37f-4904-9dc7-e555f1d3b65c)

Any suggestions on what setting(s) to change or where to look to see why it's flagging all these successful downloads as hidden? It's very disconcerting when I go looking for something that I _thought_ was downloaded only to find that it wasn't. Oh, wait... it's been hidden and it was successful...

**Comment by dnzbk** [comment]
Have any changes been made in `SETTINGS`->`DOWNLOAD QUEUE`->`KeepHistory`?

**Reply by daFreeMan** [reply]
`Keep History` is currently set to 30 days. I'm pretty sure I haven't made any changes to that, but wouldn't swear to it under oath. Even so, I wouldn't think that a 30-day window would cause something downloaded "today" to be hidden...

Not sure if it matters, but I've just realized that all the `Hidden` ones have a blank category. All the ones that remain visible have some category set. 

Not sure why they're coming down with no category - 99% are being fed from radarr or sonarr.

**Comment by daFreeMan** [comment]
Further investigation is showing that I've got `Remove Completed` set in sonarr, but not in radarr. I'll try clearing that in sonarr to see if it makes any changes.

> I'm betting this was the issue.
> I have a couple of movies that are showing normally & have a category. 
> I have a _bunch_ of TV shows w/o a category that are Hidden, and a few with a category that are _not_ hidden.
> I'll post back in a few days to see if this resolved the issue.
> Thanks, @dnzbk for getting me to dig further. You're comment doesn't seem to have pointed me in the right direction, but got me thinking properly

I've also confirmed that both DL clients have an appropriate `Category` set, so they _should_ be supplying it to nzgbet, and seem to on occasion...

**Reply by daFreeMan** [reply]
Pretty sure that was it.

Just DL'd a new file from sonarr with `Remove Complete` unchecked. It's properly labeled with a category and isn't flagged Hidden.

Happy to close this now, if that's the proper action to take.

**Comment by dnzbk** [comment]
Interesting case. 
I'm not an expert in tools like sonarr and radarr, so thank you for your investigation and I'm very glad it was successful.
